pkgs/applications/office/scribus/default.nix

```diff
@@ -44,11 +44,11 @@ in stdenv.mkDerivation rec {
     done
   '';
 
-  meta = {
-    maintainers = [ stdenv.lib.maintainers.marcweber ];
-    platforms = stdenv.lib.platforms.linux;
+  meta = with stdenv.lib; {
```

Comment:
How does this changes for scribus have anything to do with qt515? Seems they are already using qt515:

https://github.com/NixOS/nixpkgs/blob/f0faa10d8a29c16dfbbecd76f2d974915cd9728c/pkgs/top-level/all-packages.nix#L24033-L24037

Probably a merge conflict resolution issue.

Reply:
Oh yes, strange, I dont't remember what happened I remove this commit